### PR TITLE
generate random IDs, and store 'current' ID

### PR DESCRIPTION
**Pull request by glennhartmann**
One more step towards supporting multiple images.

### CODE REVIEW

File: src/storage.ts [hunk 1]
```diff
@@ -15,9 +15,14 @@
  */
 
 import { openDB, DBSchema, IDBPDatabase } from 'idb';
+import { v4 as uuidv4 } from 'uuid';
```
Comment (beverloo) [inline src/storage.ts:18]: What is the bundle size increase we're seeing by including this dependency?

File: src/storage.ts [hunk 2]
```diff
@@ -15,9 +15,14 @@
...
 
 const kMainDBName = 'appDB';
-export const kMainImageName = 'mainImage';
+
+// kCurrentBlobIDKey is the key into the miscStringVals store for the
+// FileUniqueID of the "current" blob (ie, the one that will be displayed
+// when the app is first opened).
+const kCurrentBlobIDKey = 'currentBlobID';
```
Comment (beverloo) [inline src/storage.ts:25]: Will we remove this again when we have proper multi-file support?

**Review by beverloo — APPROVED** [review]

File: src/storage.ts (hunk 1 above)
Comment (glennhartmann) [reply src/storage.ts:18]: `llaminator.<hash>.js`: 81,100 B -> 82,589 B (~1.45 KiB, ~1.84% increase)
`sw.js`: 20,761 B -> 22,099 B (~1.31 KiB, ~6.44% increase)

File: src/storage.ts (hunk 2 above)
Comment (glennhartmann) [reply src/storage.ts:25]: Quite possibly, but I think that depends on the UX we land on.

Off the top of my head, the two obvious directions we could go are:

1. launch into a "gallery" or image picker sort of UI, where the user picks which image to show (possibly with a special case to skip straight to the image if only 1 exists), or
2. launch into a single (or a small amount of?) full image[s] based on either which one[s] were displayed most recently or which one[s] the user has chosen to "pin"

If we go with some variation of the latter option, then we'll still need to have some concept of the "current" or "pinned" image[s] to display as soon as the app is launched

**Pull request merged**